engine/schema/src/main/resources/META-INF/db/schema-41510to41600.sql

```diff
@@ -19,21 +19,293 @@
 -- Schema upgrade from 4.15.1.0 to 4.16.0.0
 --;
 
--- Adding dynamic scalable flag for service offering table
-ALTER TABLE `cloud`.`service_offering` ADD COLUMN `dynamic_scaling_enabled` tinyint(1) unsigned NOT NULL DEFAULT 1  COMMENT 'true(1) if VM needs to be dynamically scalable of cpu or memory';
-DROP VIEW IF EXISTS `cloud`.`service_offering_view`;
-CREATE VIEW `cloud`.`service_offering_view` AS
+
+ALTER TABLE `cloud`.`service_offering` ADD COLUMN `uuid` varchar(40) UNIQUE;
+ALTER TABLE `cloud`.`service_offering` ADD COLUMN `name` varchar(255);
+ALTER TABLE `cloud`.`service_offering` ADD COLUMN `display_text` varchar(4096);
```

Comment:
DEFAULT NULL for default text (keep consistent with _disk_offering_ table details)